script.js

```diff
@@ -1,4 +1,117 @@
+// Define current player
+var currentPlayer = 1;
+
+// Function for output
+var myOutputValue = "";
+
+// Define variables to keep track of player's number
+var playerNumber = [];
+var playerOneNum = playerNumber[0];
+var playerTwoNum = playerNumber[1];
+
+// Function to track game mode
+var modeRollDice = "Roll dice mode";
+var modeChoseOrder = "Chose order of numbers";
+var modeCurrent = modeRollDice;
+
+// Function to keep track of dice rolled
+var diceNumberRolled = [];
+var diceRollA = "";
+var diceRollB = "";
+
+// Combination of numbers
+// var combineNumbers = "";
+var combineNumbersAB = Number(diceRollA) + Number(diceRollB);
+var combineNumbersBA = Number(diceRollB) + Number(diceRollA);
+
+// Function to get random dice number
+var rollDice = function () {
+  var randomDecimal = Math.random() * 6;
+  var randomInteger = Math.floor(randomDecimal);
+  var diceNumber = randomInteger + 1;
+  return diceNumber;
+};
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  // Select game mode
+  var gameCounter = 0;
+  if (modeCurrent == modeRollDice) {
+    myOutputValue = rollDiceTwice(gameCounter);
```

Comment:
`myOutputValue` here will be using the version in the global scope, while the version in [line 46](https://github.com/rocketacademy/basics-beat-that/blob/e3e941a6585a42407eb2a4218d945119427bf058/script.js#L46) is in the function scope.

How about creating `myOutputValue` only within the `main` function so that it is scoped here? You can create it within the function outside of the `if` blocks that check for the game states, such as [below line 35](https://github.com/rocketacademy/basics-beat-that/blob/e3e941a6585a42407eb2a4218d945119427bf058/script.js#L36).

You can go `var myOutputValue = ...` below line 35, and just re-assigning strings to it to change its strings.